R/twidlr-e1071.R

```diff
@@ -33,3 +33,48 @@ predict.svm <- function(object, data, ...) {
   data <- predict_checks(data = data, ...)
   e1071:::predict.svm(object, newdata = data, ...)
 }
+
+
+#' data.frame-first formula-second method for \code{\link[e1071]{naiveBayes}}
+#'
+#' This function passes a data.frame, formula, and additional arguments to
+#' \code{\link[e1071]{naiveBayes}}.
+#'
+#' @seealso \code{\link[e1071]{naiveBayes}}
+#'
+#' @inheritParams twidlr_defaults
+#' @export
+#'
+#' @examples
+#'
+#' data <- mtcars
+#' data$cyl_gr_6 <- ifelse(data$cyl > 6, 1, 0)
+#'
+#' fit <- naiveBayes(data, cyl_gr_6 ~ .)
+#' summary(fit)
+#'
+#' # Help page for function being twiddled
+#' ?e1071::naiveBayes
+#'
+naiveBayes <- function(data, formula, ...) {
+  check_pkg("e1071")
+  UseMethod("naiveBayes")
+}
+
+#' @export
+naiveBayes.default <- function(formula, data, ...) {
```

Comment:
Please follow new convention of using `coerce_args()` to coerce `data` and `formula` appropriately. For example, see https://github.com/drsimonj/twidlr/blob/master/R/twidlr-e1071.R#L21-L27